engine-precompiles/src/native.rs

```diff
@@ -237,17 +244,26 @@ impl<I: IO> Precompile for ExitToNear<I> {
         context: &Context,
         is_static: bool,
     ) -> EvmPrecompileResult {
+        // ETH transfer input format: (21+ bytes)
+        //  - flag (1 byte)
+        //  - refund_address (20 bytes)
+        //  - 1 byte ??
+        //  - recipient_account_id (N bytes)
+        // ERC20 transfer input format: (54+ bytes)
+        //  - flag (1 byte)
+        //  - refund_address (20 bytes)
+        //  - 1 byte ???
+        //  - amount (32 bytes)
+        //  - recipient_account_id (N bytes)
+        is_valid_input_size(input, 21, None)?;
```

Comment:
It seems that this call should be in the body of `parse_input` function?

Reply:
Which one? There are 2 functions with the same name.